Automattic/jetpack
# icons: update Loom icon according to new branding

This PR updates the Loom icon, according to the request asked in #17479.

> We'd really appreciate it if you could swap out the current logo in the editor with the new one, attached to this issue in .svg.

Fixes https://github.com/Automattic/jetpack/issues/17479

#### Changes proposed in this Pull Request:

* Update embed Loom variation icon

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Apply these changes
* Insert a new Loom block
* Check the new design

Before | After
----|----
![image](https://user-images.githubusercontent.com/77539/96003489-a676c800-0e10-11eb-819f-9fb6892486b4.png) | ![image](https://user-images.githubusercontent.com/77539/96003408-94952500-0e10-11eb-9228-962c5af1cec5.png)


#### Proposed changelog entry for your changes:

* Update embed Loom variation icon
